pkg/objstore/swift/swift.go

```diff
@@ -12,6 +12,7 @@ import (
 	"time"
 
 	"github.com/improbable-eng/thanos/pkg/objstore"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
please remove the blank line after this one

Reply:
ran `goimports` again

Reply:
goimports doesn't solve the blank line issue, please remove the blank line after this

Reply:
i meant removed blank line and did the goimports again